glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/MetricsPingScheduler.kt

```diff
@@ -205,6 +186,19 @@ internal class MetricsPingScheduler(
      */
     fun schedule(overduePingAsFirst: Boolean) {
```

Comment:
We're no longer calling `schedule()` from the lifecycle observer, I think we can get rid of the `overduePingAsFirst` and always assume that `schedule` is called from the Glean init path. We should also change the function docs accordingly.

Reply:
Good idea.  This removes even more lines of code!